src/components/parameter/ParameterEdit.vue

```diff
@@ -124,19 +124,30 @@
 
             <template v-if="state.options.input === 'tickSlider'">
               <v-subheader class="paramLabel" v-text="label()" />
-              <v-slider
-                :max="state.options.ticks.length - 1"
-                :thumb-color="state.color"
-                :tick-labels="state.options.ticks"
-                @change="paramChange"
-                class="mb-1"
-                dense
-                height="40"
-                hide-details
-                ticks="always"
-                tick-size="4"
-                :value="state.value"
-              />
+              <v-tooltip left v-model="state.showResolutionWarning">
+                <template v-slot:activator="{ on, attrs }">
+                  <v-slider
+                    :max="state.options.ticks.length - 1"
+                    :thumb-color="state.color"
+                    :tick-labels="state.options.ticks"
+                    @change="paramChange"
+                    class="mb-1"
+                    dense
+                    height="40"
+                    hide-details
+                    ticks="always"
+                    tick-size="4"
+                    :value="state.value"
+                    v-bind="attrs"
+                    v-on="on"
+                  />
+                </template>
+                <span>
+                  <v-icon v-text="'mdi-alert-outline'" color="white" />
+                  Beware: Small resolution values produce many datapoints, which
+                  can cause a high system load and thus freezes and lags!</span
+                >
```

Comment:
```suggestion
                  Beware: Small resolution values produce many datapoints,
                  <br>
                  which might cause a high system load and thus freezes and lags!
                </span>          
```

Reply:
Good point, I added it with the commit below.